### PR TITLE
14

### PR DESCRIPTION
**Pull request by osaosula**
Add new patient UI

### CODE REVIEW

File: src/pages/AddPatient.jsx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+              <Grid size={3} item>
+                Last Name
+                <TextField
+                  type="text"
+                  label="Last Name"
+                  size="small"
+                  fullWidth
+                  required
+                />
+              </Grid>{" "}
```
Comment (BorisLaboratory) [inline src/pages/AddPatient.jsx:68]: There is unwanted code block

File: src/pages/AddPatient.jsx [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+              <Grid size={3} item>
+                Email
+                <TextField
+                  label="Email"
+                  type="email"
+                  size="small"
+                  fullWidth
+                  required
+                />
+              </Grid>{" "}
```
Comment (BorisLaboratory) [inline src/pages/AddPatient.jsx:78]: There is unwanted code block

File: src/pages/AddPatient.jsx [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+              <Grid size={3} item>
+                Contact Last Name
+                <TextField
+                  type="text"
+                  label="Last Name"
+                  size="small"
+                  fullWidth
+                  required
+                />
+              </Grid>{" "}
```
Comment (BorisLaboratory) [inline src/pages/AddPatient.jsx:157]: There is unwanted code block

**Review by BorisLaboratory — CHANGES_REQUESTED** [review]
Looks good but with a few observations . See comments

File: src/pages/AddPatient.jsx [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+              <Grid size={3} item>
+                First Name
+                <TextField
+                  label="First Name"
+                  type="text"
+                  size="small"
+                  fullWidth
+                  required
+                />
+              </Grid>{" "}
```
[the remark this answers is not in the document]
Comment (osaosula) [reply src/pages/AddPatient.jsx:58]: Fixed

File: src/pages/AddPatient.jsx (hunk 1 above)
Comment (osaosula) [reply src/pages/AddPatient.jsx:68]: Fixed

File: src/pages/AddPatient.jsx (hunk 2 above)
Comment (osaosula) [reply src/pages/AddPatient.jsx:78]: Fixed

File: src/pages/AddPatient.jsx (hunk 3 above)
Comment (osaosula) [reply src/pages/AddPatient.jsx:157]: Fixed

File: src/pages/AddPatient.jsx [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+                Relationship to Patient
+                <TextField
+                  label="Relationship"
+                  placeholder="Relationship"
+                  type="text"
+                  size="small"
+                  fullWidth
+                  required
+                />
+              </Grid>{" "}
```
[the remark this answers is not in the document]
Comment (osaosula) [reply src/pages/AddPatient.jsx:168]: Fixed

File: src/pages/AddPatient.jsx [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+              <Grid size={3} item>
+                Contact First Name
+                <TextField
+                  label="First Name"
+                  type="text"
+                  size="small"
+                  fullWidth
+                  required
+                />
+              </Grid>{" "}
```
[the remark this answers is not in the document]
Comment (osaosula) [reply src/pages/AddPatient.jsx]: Fixed

File: src/pages/AddPatient.jsx [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+                  type="email"
+                  size="small"
+                  fullWidth
+                  required
+                />
+              </Grid>{" "}
+              <Grid size={3} item>
+                Mobile
+                <TextField
+                  label="Mobile No"
```
[the remark this answers is not in the document]
Comment (osaosula) [reply src/pages/AddPatient.jsx:82]: Fixed

**Review by BorisLaboratory — APPROVED** [review]
Looks good

**Pull request merged**